src/Clapi/TextSerialisation.hs

```diff
@@ -0,0 +1,33 @@
+module Clapi.TextSerialisation where
+import Data.Monoid
+import qualified Data.Map as Map
+import Blaze.ByteString.Builder (Builder)
+import Blaze.ByteString.Builder.Char.Utf8 (fromString, fromChar)
+import qualified Clapi.Serialisation as Wire
+import Clapi.Types (Time(..), ClapiValue(..))
+
+typeTag :: ClapiValue -> Char
+typeTag (ClBool _) = 'B'
+typeTag v = Wire.typeTag v
+
+fss :: Show a => a -> Builder
+fss a = fromString $ show a
+
+cvBuilder :: ClapiValue -> Builder
+cvBuilder (ClBool True) = fromChar 'T'
+cvBuilder (ClBool False) = fromChar 'F'
+cvBuilder (ClInt32 i) = fss i
+cvBuilder (ClString s) = fss s
```

Comment:
Presumably `ghc` will moan at you about incomplete pattern matches if you `-Wall`?